.circleci/config.yml

```diff
@@ -41,7 +41,7 @@ jobs:
       - run:
           name: Integration tests compile
           working_directory: ~/workbench/api
-          command: ./project.rb gradle compileBigQuery compileIntegration
+          command: ./project.rb gradle compileBigquerytestJava compileIntegrationJava
```

Comment:
why is the t in test lowercase? Likewise, why not have a `Test` after `Integration`?

Reply:
not sure. These tasks were auto generated. It did worry me a little that these had to be changed but the builds are passing.

Reply:
Do you mind double checking what changed? There may be things that silently broke if name conventions got switched.

Reply:
Looked into it.

Looks like the gradle Java plugin auto generates `compile___Java` tasks for all source sets, `bigquerytest` being one of ours. https://github.com/all-of-us/workbench/blob/master/api/build.gradle#L273
https://docs.gradle.org/current/userguide/java_plugin.html#java_source_set_tasks

This generates the task `compileBigquerytestJava` but because gradle supports auto completion of tasks, it was able to map `compileBigQuery` -> `compileBigquerytestJava` even though `compileBigQuery` technically does not exist. 

Adding the Kotlin plugin generates another set of `compile___Kotlin` tasks for all source sets. At this point, gradle can no longer auto complete `compileBigQuery` since there are two possibilities with `compileBigquerytestJava` and `compileBigquerytestKotlin`.


I looked through the rest of our codebase and I couldn't find other instances where the addition of new gradle tasks would cause auto completion ambiguity.

Reply:
Thanks for chasing that down. This is an odd convention having a prefix and suffix like that. Is it possible to have something horrendous like `compileBigQuery_test_Java` or something to distinguish the parts?